src/components/App.tsx

```diff
@@ -597,6 +609,9 @@ class App extends React.Component<ExcalidrawProps, AppState> {
     }
 
     if (isCollaborationScene) {
+      // when joining a room we don't want user's local scene data to be merged
+      //  into the remote scene
+      this.resetScene();
       this.initializeSocketClient({ showLoadingState: true });
```

Comment:
How many callsites do we have of this? Should we reset the scene when we show loading state to true?